reflex/reflex.py

```diff
@@ -255,6 +269,38 @@ def export(
         )
 
 
+db_cli = typer.Typer()
+
+
+@db_cli.command()
+def migrate():
+    """Create or update database schema based on app models or existing migration scripts."""
+    # explicitly running `reflex db migrate` overrides config file value
+    get_config().db_migrate = True
+    prerequisites.get_app()
+    prerequisites.check_schema_up_to_date()
+
+
+@db_cli.command()
+def makemigrations():
+    """Create autogenerated alembic migration scripts."""
+    if not model.has_alembic:
+        console.print(
+            "[red]Alembic is not installed. Run [bold]pip install alembic[/bold] first."
```

Comment:
I'm not sure if we should prompt the user to specifically use pip. They might be using something else eg. poetry or pipenv. Maybe we could have a more generic message here?

Reply:
alembic was made part of the requirements, and will be present whenever reflex is installed.

down the road, we might pull the db stuff into an optional extra, and alembic will be part of that extra (think `pip install reflex[db]`)